osbuild/pipeline.py

```diff
@@ -131,11 +130,11 @@ def run(self, tree, runner, build_tree, monitor, libdir, output_dir=None, var="/
             }
 
             binds = []
-            if output_dir:
-                output_dir = os.fspath(output_dir)
-                os.makedirs(output_dir, exist_ok=True)
-                binds.append(f"{output_dir}:/run/osbuild/output")
-                args["output_dir"] = "/run/osbuild/output"
+            
```

Comment:
pylint complains there is trailing whitespace in that line. Github-UI does not show it, I am unsure what that is all about. Btw., you can run `make test-src` to run quick source-code checks locally and catch pylint warnings just like the CI does. `make test-module` runs quick unit-tests. The other stuff (`test-run` and `test-data`) takes quite a long time, though.